src/project.cc

```diff
@@ -307,13 +312,8 @@ std::vector<Project::Entry> LoadCompilationEntriesFromDirectory(
     // LOG_S(INFO) << "Got args " << StringJoin(entry.args);
 
     our_time.Resume();
-    std::string absolute_filename;
```

Comment:
Why did you change entry.file to be relative? I think this should stay absolute, so ie, reading the file before passing to clang work as expected. If I revert this part of the change it seems to work fine on chrome.

Reply:
I suspect this patch assumes that cquery working dir is the same as the project working dir (which is not always true, ie, in the case of vscode).

Reply:
Reverted this change.